src/frontend/AppNavigator.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import {NavigatorScreenOptions, RootStack} from './Navigation/Stack';
+import {useIntl} from 'react-intl';
+// import {SecurityContext} from '../context/SecurityContext';
+
+import BootSplash from 'react-native-bootsplash';
+import {useDeviceInfo} from './hooks/server/deviceInfo';
+import {Loading} from './sharedComponents/Loading';
+import {createDeviceNamingScreens} from './Navigation/Stack/DeviceNamingScreens';
+import {usePrefetchLastKnownLocation} from './hooks/useLastSavedLocation';
+import {usePersistedDraftObservation} from './hooks/persistedState/usePersistedDraftObservation';
+import {usePresetsQuery} from './hooks/server/presets';
+import {initializeInviteListener} from './initializeInviteListener';
+import {ProjectInviteBottomSheet} from './sharedComponents/ProjectInviteBottomSheet';
+import {getInitialRouteName} from './utils/navigation';
+import {createDefaultScreenGroup} from './Navigation/Stack/AppScreens';
+
+// Note that this does the same things as the strange syntax found in
+// https://github.com/react-navigation/react-navigation/issues/9578#issuecomment-1022991270
+// `{createScreens()}` is equivalent to `{(() => createScreens())()}`
+// Note that screen groups should have a `key` prop, so that React knows how to update them efficiently.
```

Comment:
Is this supposed to be somewhere else?

```suggestion
```

Reply:
https://github.com/digidem/comapeo-mobile/pull/313/commits/f97294fde2f55bc471f2a5538c5569e0b1262af0